### PR TITLE
Changed the way index pattern are refreshed

### PR DESCRIPTION
**Pull request by pablomarga**
Hi teams this PR solves a problem that was caused due to before to refresh the fields of the index pattern we used an endpoint that was removed.

### CODE REVIEW

File: public/react-services/pattern-handler.js [hunk 1]
```diff
@@ -111,15 +111,12 @@ export class PatternHandler {
    */
   static async changePattern(selectedPattern) {
     try {
+      let prueba = AppState.getCurrentPattern();
```
Comment (frankeros) [inline public/react-services/pattern-handler.js:114]: remove this

File: public/react-services/pattern-handler.js [hunk 2]
```diff
@@ -111,15 +111,12 @@ export class PatternHandler {
...
-      await GenericRequest.request(
-        'GET',
-        `/elastic/known-fields/${selectedPattern}`,
-        {}
-      );
+      await this.refreshIndexPattern();
       return AppState.getCurrentPattern();
     } catch (error) {
-      throw new Error('Error Pattern Handler (changePattern)');
+      throw new Error('Error Pattern Handler (changePattern)',error);
```
Comment (frankeros) [inline public/react-services/pattern-handler.js:119]: new Error only gets one parameter for the message.

**Review by frankeros — CHANGES_REQUESTED** [review]

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by frankeros — APPROVED** [review]
LGTM!

**Pull request merged**